pkg/controller/jaeger/configmap.go

```diff
@@ -63,3 +67,30 @@ func (r *ReconcileJaeger) applyConfigMaps(ctx context.Context, jaeger v1.Jaeger,
 
 	return nil
 }
+
+func (r *ReconcileJaeger) getSelectorsForConfigMaps(instanceName string, components []string) labels.Selector {
+	nameReq, _ := labels.NewRequirement("app.kubernetes.io/name", selection.DoubleEquals, []string{util.Truncate(instanceName, 63)})
```

Comment:
Document why you are swallowing the error.

Reply:
Done